table/queries/derived_satellite_scans.sql

```diff
@@ -129,7 +129,15 @@ CREATE TEMP FUNCTION OutcomeString(domain_name STRING,
                 WHEN (SELECT LOGICAL_OR(answer.matches_control.asn)
                       FROM UNNEST(answers) answer)
                       THEN "✅answer:matches_asn"
-                ELSE CONCAT("❓answer:not_validated:", AnswersSignature(answers))
+                # TODO: delete, this case is covered by all the cert cases above
+                WHEN (SELECT LOGICAL_OR(NOT (a.https_response_status IS NULL))
```

Comment:
Can we tell whether we are able to establish a TCP connection to port 443?

I would suggest to create a "{error}:{AS}", and move it to right after `matches:ip`. Possible errors: connection_timeout, address_unreachable, network_unreachable, connection_refused.
Example: `connection_timeout:TWITTER` or maybe `answer:TWITTER:connection_refused`

We would still keep the not_validated as an unexpected catch all in that case.

Reply:
BTW, you can find the syscall error strings from Go at https://cs.opensource.google/go/go/+/master:src/syscall/zerrors_linux_amd64.go